### DISCUSSION TITLE
Draw a non-filled circle

### DISCUSSION THREAD

**Discussion by themanh01** [pokepetter/ursina#321] [Q&A]
How to draw a non-filled circle?
I tried Entity(model=Circle(mode='line')) but got this error:
File "C:\Users\Admin\AppData\Local\Programs\Python\Python39\Lib\site-packages\ursina\models\procedural\circle.py", line 16, in __init__
    self.vertices.append(verts[0])
NameError: name 'verts' is not defined
Any body help

**Comment by pokepetter** [comment]
This got fixed a while ago. Try updating.

**Reply by themanh01** [reply]
Thanks for your replying. Have a good day!!!
